x-pack/platform/plugins/shared/security/server/security_checkup/check_cluster_data.test.ts

```diff
@@ -113,7 +113,7 @@ function mockIndicesStatsResponseFactory(
 
   listOfIndicesWithCount.forEach((indexPair) => {
     result!.indices![indexPair.name] = {
-      total: { docs: { count: indexPair.count } },
+      total: { docs: { count: indexPair.count, total_size_in_bytes: 0 } },
```

Comment:
Would there be any value add by including the ability to customize the total size in this mock?